src/main/java/seedu/address/ui/UpcomingDateCard.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.model.person.PersonEvent;
+
+public class UpcomingDateCard extends UiPart<Region> {
+
+    private static final String FXML = "UpcomingDateCard.fxml";
+
+    public final PersonEvent personEvent;
+
+    @FXML
+    private VBox dateCardPane;
+
+    @FXML
+    private Label description;
+
+    @FXML
+    private Label date;
+
+    /**
+     * Creates a {@code UpcomingDateCard} with the given {@code PersonEvent}.
+     * @param personEvent
+     */
+    public UpcomingDateCard(PersonEvent personEvent) {
+        super(FXML);
+        this.personEvent = personEvent;
+        description.setText(personEvent.getDescription());
+        date.setText(personEvent.getDate() + "/" + personEvent.getMonth());
```

Comment:
Can we have `date.setText(personEvent.toUi())`? Makes it easy to test the formatting later on